constraints-dev.txt

```diff
@@ -85,7 +85,7 @@ sphinxcontrib-images==0.7.0
 sphinxcontrib-websupport==1.1.0
 treq==18.6.0
 Twisted==18.9.0
-typed-ast==1.1.2
+typed-ast==1.3.1
```

Comment:
why the change? What is it required for?

Reply:
This was updated when I upgraded mypy itself, so I guess it's a dependency of mypy.

Reply:
I see. Makes sense :+1: